ex03.exs

```diff
@@ -46,25 +51,61 @@ defmodule Ex03 do
 
   35 points:
      it works and passes all tests:    25
-     it contains no conditional logic:  3
+     it contains no conditional logic:  3       WOOOOO!!!! I got 3 points at least!!!!
      it is nicely structured            7
   """
 
+  def split(collection, process_count) do
+    tempList = Enum.to_list(collection) #converts collection to a list
+    num = div(List.last(tempList), process_count) + 1 #num = the size of each of the sublists(add one in case uneven split)
```

Comment:
Actually the +1 fails when the division is exact. It should be `div(Enum.count(coll) + process_count - 1, process_count)`